packages/connect/e2e/__fixtures__/getFeatures.ts

```diff
@@ -22,253 +23,278 @@ const customFirmwareBuild =
-                model: expect.any(String), // "T" | "R"
-                internal_model: expect.any(String),
-                fw_major: null,
-                fw_minor: null,
-                fw_patch: null,
-                // fw_vendor: null, // "EMULATOR" since 2.5.1
-                unfinished_backup: expect.any(Boolean),
-                no_backup: expect.any(Boolean),
-                recovery_mode: false,
-                capabilities: expect.arrayContaining([
-                    'Capability_Bitcoin',
-                    'Capability_Bitcoin_like',
-                    'Capability_Binance',
-                    'Capability_Cardano',
-                    'Capability_Crypto',
-                    'Capability_Ethereum',
-                    'Capability_Monero',
-                    'Capability_Ripple',
-                    'Capability_Stellar',
-                    'Capability_Tezos',
-                    'Capability_U2F',
-                    'Capability_Shamir',
-                    'Capability_ShamirGroups',
-                    'Capability_PassphraseEntry',
-                    'Capability_NEM',
-                    'Capability_EOS',
-                ]),
-                backup_type: 'Bip39',
-                sd_card_present: expect.any(Boolean), // T2T1 true, T2B1 false
-                sd_protection: false,
-                wipe_code_protection: false,
-                session_id: expect.any(String),
-                passphrase_always_on_device: false,
-            },
-            legacyResults: [
-                {
-                    rules: ['<2.4.2'], // 2.4.2 removed Lisk capability
-                    success: true,
-                    payload: {
-                        capabilities: expect.arrayContaining([
-                            'Capability_Bitcoin',
-                            'Capability_Bitcoin_like',
-                            'Capability_Binance',
-                            'Capability_Cardano',
-                            'Capability_Crypto',
-                            'Capability_EOS',
-                            'Capability_Ethereum',
-                            'Capability_Lisk',
-                            'Capability_Monero',
-                            'Capability_NEM',
-                            'Capability_Ripple',
-                            'Capability_Stellar',
-                            'Capability_Tezos',
-                            'Capability_U2F',
-                            'Capability_Shamir',
-                            'Capability_ShamirGroups',
-                            'Capability_PassphraseEntry',
-                        ]),
-                        session_id: expect.any(String),
-                        passphrase_always_on_device: false,
-                    },
+const tests: Array<{
+    description: string;
+    skip: string[];
+    params: FeaturesParams;
+    result: Features;
+    legacyResults: Array<{
```

Comment:
I don't like that much introducing this type here without connecting it to other places that actually consume it. 
on the other hand it doesn't hurt anybody, just that I would like to have it solved everywhere in one blow. 

feel free to ignore this. :D

Reply:
This `legacyResults` are a bit edge case, the reason I am adding types to fixtures in general is to ease the changes in the future. Currently, typescript wont warn you, and you have to fix failing tests one by one a figuring out whats wrong, which is time-consuming